packages/flutter/test/widgets/draggable_scrollable_sheet_test.dart

```diff
@@ -1209,4 +1209,18 @@ void main() {
     expect(controller.isAttached, true);
     expect(controller.size, isNotNull);
     });
+
+    testWidgets('DraggableScrollableController.animateTo should not leak Ticker', (WidgetTester tester) async {
```

Comment:
nit: add a comment for the relation issue
// Regression test for https://github.com/flutter/flutter/issues/102483